cetlvast/suites/docs/examples/example_07_polymorphic_alloc_deleter.cpp

```diff
@@ -148,3 +228,34 @@ TEST(example_07_polymorphic_alloc_deleter, example_usage_1)
 
     //![example_usage_1]
 }
+
+TEST_F(example_07_polymorphic_alloc_deleter, example_usage_2)
+{
+    //![example_usage_2]
+
+    cetl::pmr::polymorphic_allocator<MyObject> alloc{cetl::pmr::new_delete_resource()};
+
+    auto obj0 = cetl::pmr::InterfaceFactory::make_unique<MyObject>(alloc, "obj0", 4U);
```

Comment:
This is as it was before (with `cetl::pmr::Factory`) - we make `MyObject` instance, and return `MyObject` "interface".